srcpkgs/python3-pyside2/template

```diff
@@ -18,34 +18,57 @@ makedepends="python3-devel libshiboken2-devel qt5-devel
  qt5-serialport-devel
  $(vopt_if webengine 'qt5-webengine-devel')"
 depends="python3-shiboken2"
+checkdepends="python3-distro tzdata pulseaudio qt5-quickcontrols"
 short_desc="Python3 bindings for the Qt5 toolkit"
 maintainer="yopito <pierre.bourgin@free.fr>"
 license="LGPL-3.0-or-later"
 homepage="https://wiki.qt.io/Qt_for_Python"
 distfiles="https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-${version}-src/${_pkgname}.tar.xz"
-checksum=f1cdee53de3b76e22c1117a014a91ed95ac16e4760776f4f12dc38cd5a7b6b68
+checksum=b306504b0b8037079a8eab772ee774b9e877a2d84bab2dbefbe4fa6f83941418
+
+build_options="webengine"
+desc_option_webengine="Build Qt5 WebEngine bindings"
+
+subpackages="libpyside2-python3-devel libpyside2-python3"
 
 # needed by shiboken2 runtime
-export CLANG_INSTALL_DIR=${XBPS_CROSS_BASE}/usr
+export CLANG_INSTALL_DIR=/usr
 
 if [ ${CROSS_BUILD} ]; then
 	hostmakedepends+=" qt5-qmake qt5-host-tools qt5-tools qt5-remoteobjects qt5-scxml"
-fi
 
-build_options="webengine"
-desc_option_webengine="Build Qt5 WebEngine bindings"
+	# make clang of shiboken aware of crossbuild
+	SHIBOKEN_CLANG_OPTIONS="--target=${XBPS_CROSS_TRIPLET}"
+	SHIBOKEN_CLANG_OPTIONS+=" --sysroot=/usr/${XBPS_CROSS_TRIPLET}"
 
-if [ "$XBPS_TARGET_ENDIAN" = "le" ]; then
-	build_options_default="webengine"
+	# qt5 is built by gcc, so clang of shiboken needs gcc stuff, at least for armv7l
+	if [ "$IN_CHROOT" ]; then
+		local gcc_ver="$(${XBPS_CROSS_TRIPLET}-gcc -dumpversion)"
+		local inc_root="/usr/${XBPS_CROSS_TRIPLET}/usr/include/c++/${gcc_ver}"
+
+		SHIBOKEN_CLANG_OPTIONS+=" -isystem ${inc_root}"
+		SHIBOKEN_CLANG_OPTIONS+=" -isystem ${inc_root}/${XBPS_CROSS_TRIPLET}"
+		# XXX uneeded so far
+		#SHIBOKEN_CLANG_OPTIONS+=" -isystem ${inc_root}/backward"
+	fi
```

Comment:
I don't get why `IN_CHROOT`... No other template uses it, either. Does it need to be inside a conditional?

Reply:
should be in `CROSS_BUILD`, no?

Reply:
@ericonr  : Because gcc is not available until in chroot, but it can be removed if your prefer.
@q66 : it's within crossbuild condition.

Reply:
Ah, I see. Maybe we can update the value from within `pre_configure`, then? I think it'd work better.

Reply:
IMO, it should be wrapped inside `pre_configure` and `pre_build` instead of put in global scope, we can remove `$IN_CHROOT` then

Reply:
ah, thanks @sgn for reminding about pre_ stages.  
I had trouble to export variable `SHIBOKEN_CLANG_OPTIONS`, hence my quirk above ... 
fixed by use of `pre_build`, let's GH actions processing it again.